### PR TITLE
SPA Quiz ready for review

### PR DESCRIPTION
**Pull request by shupac800**

### CODE REVIEW

File: javascripts/iife-1.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      outputStr += `<div class="col-md-2 heading">Model</div>`;
+      outputStr += `<div class="col-md-2 heading">Color</div>`;
+      outputStr += `<div class="col-md-3 heading">Description</div>`;
+      outputStr += `<div class="col-md-1 heading">Price</div>`;
+      outputStr += `<div class="col-md-1 heading">Sold</div>`;
+      outputStr += '</div>';
+
+      for (var i = 0; i < inventory.length; i++) {
+        // output one row per car
+        outputStr += `<div class="row car-row" style="border: 2px solid ${inventory[i].color}">`;
```
Comment (C-Stein) [inline javascripts/iife-1.js]: Make sure you follow the directions exactly (Loop over your array of cars and build up an HTML string that will create a series of Bootstrap rows with 3 columns in each one. )

File: javascripts/iife-2.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+all of the eventHandlers that you need for the application. Name the function 
+activateEvents.*/
+
+
+// this IIFE augments the class CarLot created in iife-1.js
+var CarLot = (function(newCarLot) {
+
+  var currentlySelectedRow;  // this could also go inside activateEvents function
+
+  // PUBLIC FUNCTION:  add event handlers
```
Comment (C-Stein) [inline javascripts/iife-2.js]: nice use of comments!

**Comment by C-Stein** [comment]
Nice work! You do need to fix one thing (see the comment on line 18 in IIFE-1) before you'll be ready to merge, though, just to make sure you're following the directions exactly.

File: javascripts/iife-1.js (hunk 1 above)
Comment (shupac800) [inline javascripts/iife-1.js]: I saw that "3 columns" thing in the directions and didn't know what to make
of it.  There are 7 data fields for each car -- if "3 columns" is intended
to mean (a) 3 displayed columns or (b) "col-md-3" (which would make 12/3 =
4 columns), I couldn't envision how I could do either and still meet the
requirement to display all 7 categories of car data.

So, I mean, I'd be happy to fix this -- can you clarify what the spec means
by "3 columns"?

Maybe it means make three columns of col-md-4, then subdivide them to make
7 total columns?

**Comment by shupac800** [comment]
I think I understand now what the instructions wanted me to do with the Bootstrap grid.  I've made three columns of col-md-4, then subdivided them to make 7 total fields.

**Comment by shupac800** [comment]
This time for sure! Now, in iife-1.js, each set of nested columns is enclosed within a <div class='row'>.

**Pull request merged**